### DISCUSSION TITLE
Duplicated line in `docs/usage/configuration-options.md`

### DISCUSSION THREAD

**Discussion by dklimpel** [renovatebot/renovate#26476] [Request Help] [answered]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us what version of Renovate you run.

37

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

None

### Describe the problem

There is a duplicated line in docs:

https://github.com/renovatebot/renovate/blob/b4dd814d12fdf30c8d343ef480ee09098ce6684c/docs/usage/configuration-options.md?plain=1#L413-L414

The first one (L413) is obsolete.

This was introduced by #26377

### Relevant debug logs

None

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by secustor** [comment]
Thx for reporting this. 

Do you want to contribute this change?

**Reply by dklimpel** [reply, marked as answer]
I have a try:
- #26478
